homeassistant/components/august/activity.py

```diff
@@ -77,7 +77,7 @@ async def async_setup(self) -> None:
                 _LOGGER,
                 cooldown=ACTIVITY_DEBOUNCE_COOLDOWN,
```

Comment:
Previously, the cooldown only kicked in when the function was done. Because the update is now called in the background, we might need to increase the cooldown?

Reply:
Will do some testing to tomorrow. Marking as draft in the mean time

Reply:
Looks like it's ok and the update takes less than 100ms in most cases.  

It makes more sense to have a way to tell the debouncer to create background tasks since we are going to identify more places where the debouncer is used where we don't want task creation to delay startup

Reply:
Another place we have this problem is Bluetooth since any update that happens to happen from an advertisement that triggers a poll will block start up

Reply:
That case is much worse though since sometimes the Bluetooth connection can take 30 seconds to establish

Reply:
https://github.com/home-assistant/core/pull/113128